### ISSUE TITLE
Cannot have tilde ~ in smartphone as I don't get it in the virtual keyboard of Element-desktop in the Android client. 

### ISSUE THREAD

**Issue by shirishag75** [vector-im/element-web#22683]
### Steps to reproduce

1. I have been using element-desktop (previously known as riot) for more than 2-3 years on the desktop. 
2. When using for the password, I decided to use tilde ~ as one of the special characters as it is not used by most average people. 
3. On the desktop while signing up, was able to use my relatively strong password without any issue, login, have chats, and come back out. 
4. Just a few days ago, got a new phone, updated to Android 12, and then installed Telegram 1.4.25 which was released just a couple of days ago. 
5. Now I have a matrix client so wanted to use the same credentials on the smartphone so could use the same contacts and everything. But now I'm stuck. 

#### What did you expect?

I expected it to work without an issue. 

#### What happened instead?

I am unable to proceed at the login page itself :(

### Operating system

Android 12.

### Application version

1.4.25

### How did you install the app?

Google play store.

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by kittykat** [comment]
@shirishag75 The default Android keyboard has a `~` on the symbols view. If you are using a custom keyboard on your phone which doesn't support `~`, then it would be a good idea to try another keyboard.

If you're unsure of how to access the different views on the keyboard, there are many instructional videos available on YouTube.

I'm going to close this issue as it's not a bug in our apps.

**Comment by shirishag75** [comment]
Right. It was samsung keyboard. Once I installed the Google keyboard was able to figure out stuff.
